### ISSUE TITLE
Broadlink RM4 pro support

### ISSUE THREAD

**Issue by ndvgn** [newt10/homebridge-broadlink-rm-enhanced#11]
Is RM4 pro supported?

1) No codes are appeared in logs during learning mode.
2) No flashing light on device during switching a switch in Home App. Although the light is flashing when I send commands via Broadlink iOS app

**Comment by newt10** [comment]
Currently RM4 pro is not supported. I am looking to integrate it in the next few weeks. I don't have a device so I am also looking for help for testing. Let me know if this is something you can help with.

**Comment by ndvgn** [comment]
Thank for your reply.
I can help you with testing if you explain what to do =)
I’m an iOS developer, so I have technical skills, but I have never dealt with homebridge plugins developing.
You can write mi via Telegram: @nvdgn

**Comment by wonderbit** [comment]
I've got a RM4 mini, let me know if I can be of any help! (@wesmuller on Telegram, @wessley on Twitter)

**Comment by JacksonR64** [comment]
Been playing around with a different branch for days trying to Get my IR Fan to work more or less exactly like your branch enables but I also have a RM4 mini so it won't work. Really excited for your update and happy to help with any testing.
Twitter @TokenJako

**Comment by pickanothername** [comment]
Any updates on this @newt10? I'm looking to buy an RM4 Pro and want to know if it will be supported

**Comment by newt10** [comment]
@JuniorGenius , yes, I have been working on a rewrite of this plugin using typescript and have already added some support for RM4 devices. I can't share an exact timeline yet but I plan on supporting it.

**Comment by pickanothername** [comment]
Awesome thanks
